mods/persistence/modules/world_save/z_undefine.dm

```diff
@@ -0,0 +1,21 @@
+#undef SERIALIZER_TYPE_NULL
```

Comment:
~serializer_undefine.dm or something is preferable. The actual define file also probably doesn't need to be in a separate folder.

Reply:
I put it in a folder because folders come before files in the hierarchy.

Reply:
Mhmm, but through my brief glances, I think if you're undefining it in the same folder it's preferable to have matching files like

_world_save.dm
and
~world_save.dm

or something similar.

Reply:
Renamed